source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -1,7 +1,7 @@
 #### Order object
 
-Name         | Type   | Value | Description
------------- | ------ | ----- | -----------------
+Name         | Type | Value | Description
+------------ | -- | ----- | -----------------
```

Comment:
Is this related to this feature?

Reply:
no missed (ide automations ..) fixed aefe4c1731b3be55cf0f41436ab6260857fb4057